gnome/gnome-radio/Portfile

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gnome-radio
+version             0.1.2
+set branch          [join [lrange [split $version .] 0 1] .]
+
+categories          gnome
+platforms           darwin
+license             GPL-3+
+maintainers         {gnome.org:ole @oleaamot} \
+                    openmaintainer
+description         GNOME Radio
+long_description    Public Network Radio Software
+homepage            http://www.gnomeradio.org/
+master_sites        http://www.gnomeradio.org/${branch}/
+distfiles           ${name}-${version}.tar.gz
+checksums           rmd160   15ee087e3db1fbb3e241cafa7046f3bc3ebaadf1 \
+                    sha256  cceb733c85515dfff1daf2700d7f7c6548fe4c5431fef81272830c2cfac32168 \
+                    size    106068
+use_xz              no
+depends_build       port:autoconf \
+                    port:automake \
+                    port:geocode-glib \
+                    port:gnome-common \
+                    port:gstreamer1-gst-plugins-bad \
+                    port:gstreamer1-gst-plugins-good \
+                    port:gstreamer1-gst-plugins-ugly \
+                    port:gtk-doc \
+                    port:intltool \
+                    port:itstool \
+                    port:pkgconfig \
+                    port:yelp-tools
+
+depends_lib         path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    path:lib/pkgconfig/pango.pc:pango \
+                    port:desktop-file-utils \
+                    port:geocode-glib \
+                    port:gstreamer1 \
+                    port:gstreamer1-gst-plugins-base \
+                    port:gtk3 \
+                    port:libchamplain \
+                    port:libxml2 \
+                    port:zlib
+
+depends_run         port:adwaita-icon-theme \
+                    port:gstreamer1-gst-plugins-bad \
+                    port:gstreamer1-gst-plugins-good \
+                    port:gstreamer1-gst-plugins-ugly
+
+# reconfigure using autogen.sh from upstream git for intltool 0.51 compatibility
+
+post-patch {
+    xinstall -m 755 ${filespath}/autogen.sh ${worksrcpath}
+}
+
+configure.cmd       ./autogen.sh
```

Comment:
Instead of calling this directly, you can use (I believe) `use_autoconf yes` instead.